lib/utils.ts

```diff
@@ -10,7 +10,21 @@ export function capitalizeFirstLetter(text: string): string {
 }
 
 export const removeFilePathExtension = (filePath: string) => {
-  return filePath.split('.')[0];
+  const lastIndex = filePath.lastIndexOf('.');
+  return filePath.slice(0, lastIndex);
+};
+
+export const decodeMdxFilePathData = (
+  filePath: string,
+  lang: string
+): { id: string; urlPath: string } => {
+  const cleanPath = removeFilePathExtension(filePath);
+  const id = cleanPath.split('-')[0];
+
+  return {
+    id: id,
+    urlPath: `/${lang}/blog/${cleanPath}`,
```

Comment:
I would suggest removing the id from the urlPath. I think this way the url is cleaner, and there is no real need, as far as I know, to have the id in the final path.

Reply:
I think he's using the id to identify the blog here:
https://github.com/gironajs/gironajs.com/pull/8/files#diff-cf7e29e4902dea0aa000da86f34225ba6d7a6d13c982343f255b15f827269199R23

Reply:
<img src="https://media0.giphy.com/media/AhjXalGPAfJg4/giphy.gif"/>

Id deleted: https://github.com/gironajs/gironajs.com/pull/8/commits/b618f5b0f651f3349d93a8a343953df695475cd4